hack/lib/util/misc.sh

```diff
@@ -109,7 +109,30 @@ function os::util::format_seconds() {
 }
 readonly -f os::util::format_seconds
 
-# os::util::sed attempts to make our Bash scripts agnostic to the platform
+# os::util::sed attemps to make our Bash scripts agnostic to the platform.
+#
+# Globals:
+#  None
+# Arguments:
+#  - all: arguments to pass to `sed`
+# Return:
+#  None
+function os::util::sed() {
```

Comment:
can you move this into a utility file other then under hack/lib.  These are legacy utils which we are trying to move away from

Reply:
Moved to `hack/testing/utils`